local/bin/py/build/actions/workflows.py

```diff
@@ -37,7 +37,7 @@ def workflows(content, content_dir):
                         data = json.loads(f.read())
                     except:
                         logger.warn(f"Error parsing {file_name}")
-            if data and data.get('stability', '') == 'stable':
+            if data and data.get('stability', '') == 'stable' and not data.get('internal') and not data.get('hidden') and not data.get('deprecated'):
```

Comment:
Based on Slack convos, I think we need to move this logic down to line 44 to check the child actions

Reply:
🤦 you're absolutely right, thanks for the catch !